python/ccxt/binance.py

```diff
@@ -973,11 +1069,9 @@ def fetch_markets(self, params={}):
                     min_notional = self.safe_float(filter, 'notional')
                 entry['limits']['cost']['min'] = min_notional
 
-            if leverage_limits:
-                symbol_position_limits = self.safe_value(leverage_limits, symbol)
-                market["position_limits"] = symbol_position_limits
-                max_leverage = max([leverage_limit["leverage"]["max"] for leverage_limit in symbol_position_limits])
-                entry['limits']['leverage'] = {'max': max_leverage}
+            exists = self.handle_leverage_limits(entry, leverage_limits, market, symbol)
+            if exists is False:
+                continue
```

Comment:
Why are we continuing? if there's no leverage limit? what is filtered here? also from non-binance cross exchange?

Reply:
we're fetching all the markets inside the publicGetExchangeInfo for spot exchange and then we want to filter the non supported symbols for each exchange Binance Cross-Margin and Binance Isolated-Margin

Reply:
ah because binance has no leverage_limits, kk